### DISCUSSION TITLE
how to adjust the width of the second column?

### DISCUSSION THREAD

**Discussion by taochenshh** [mmistakes/minimal-mistakes#3090] [Q&A]
Hi, thanks for the amazing theme.

How can I change the width of the second column of this two-column style theme. For example, I wanna enlarge the width of the blog list so that each line can show more words? 

Thank you for your help.

**Comment by mmistakes** [comment]
You'll need to adjust the theme's CSS on [`.archive`](https://github.com/mmistakes/minimal-mistakes/blob/master/_sass/minimal-mistakes/_archive.scss#L5-L17) and [`.page`.](https://github.com/mmistakes/minimal-mistakes/blob/master/_sass/minimal-mistakes/_page.scss#L23-L34) If you remove the right padding it should give you the effect you want.

![image](https://user-images.githubusercontent.com/1376749/30591249-ffffa092-9d0f-11e7-8023-e362a609ba16.png)

Related: #623, #384, #74

**Comment by taochenshh** [comment]
Thanks.

**Comment by amosshi** [comment]
See #2093

**Comment by mulle-nat** [comment]
Why is it, that when I change the `right-sidebar-width` to zero in [_variables.scss](https://github.com/mmistakes/minimal-mistakes/blob/master/_sass/minimal-mistakes/_variables.scss#L158), that the left "author" column vanishes as well ?

```
$right-sidebar-width-narrow: 0px !default;
$right-sidebar-width: 0px !default;
$right-sidebar-width-wide: 0px !default;
```
That's and the background color were the only changes I made to a fork I just made. I changed the background color to a pleasant red, so I know my changes were picked up :smiley: 
 
![grafik](https://user-images.githubusercontent.com/1381995/147854174-995fb058-2472-49c9-bdeb-6e1958ba8001.png)
